fetcher/dockerhub.py

```diff
@@ -34,7 +34,7 @@ def __init__ (self, conffile, version, remote):
     def download(self):
         self.cleanworkspace()
 
-        container_name='resinhup'
+        container_name='resinos'
```

Comment:
No. It was working because there was no other container called resinhup. Meta-resin PR now uses this name.